sdk/main/src/HbarTransfer.cc

```diff
@@ -17,16 +17,16 @@
  * limitations under the License.
  *
  */
-#include "Transfer.h"
```

Comment:
I'm wondering which one is the actual version and naming for this class - in the [Java SDK](https://github.com/hashgraph/hedera-sdk-java) it's still called [Transfer](https://github.com/hashgraph/hedera-sdk-java/blob/develop/sdk/src/main/java/com/hedera/hashgraph/sdk/Transfer.java), while in [Hedera SDK Reference](https://github.com/hashgraph/hedera-sdk-reference) class `Transfer` is missing.
Shouldn't we rename it to `HbarTransfer` in the Java SDK too?

Reply:
This is an internal utility class so I'm not sure it matters. I just felt like `Transfer` was too generic now, as this PR added `TokenTransfer` and `TokenNftTransfer`.